### PR TITLE
Use actions for ci/cd

### PR DESCRIPTION
**Pull request by benmelz**
Travis is out of credits, which we can refill, but then I remembered that @bandogora is actively using actions on umts/embedded-departure-board. Possibly due to forking, but I'm just curious to see if maybe github has changed its rules to grant us actions access for our public repos?

### CODE REVIEW

File: .travis.yml
Comment (bandogora) [inline .travis.yml:1]: Should we remove this in this PR, or wait until Github CI has worked for a period of time? @werebus thoughts?

Comment (benmelz) [reply .travis.yml:1]: No, I don't see why we need to run 2 identical CI suites in parallel (and we definitely shouldn't run 2 CD jobs in parallel). Like we said, if it stops working, we revert and we're back in business (provided we have more OSS credits in travis).

The buildbot account/tokens and stuff is what should stay for a period of time though.

**Review by werebus — APPROVED** [review]
Let's see how this goes...

File: .travis.yml
Comment (bandogora) [reply .travis.yml:1]: I meant, disable travis but keep the config just in case. But yeah, you're right we can just revert.

**Comment by werebus** [comment]
Things that remain:
* Required checks
* I _think_ we can delete the `gh-pages` branch, too.

**Comment by bandogora** [comment]
I'll mention that you're file naming is a little non-standard(?). I usually see `build.yml` & `deploy.yml` but thats just bike-shedding.

It may also be worth adding
```yml
on:
  workflow_dispatch:
```
in case you need to do a manual run or testing.

**Comment by benmelz** [comment]
> * Required checks

Took me a minute to get them to show up, but all set

> * I _think_ we can delete the `gh-pages` branch, too.

Can do once we feel solid and we also clean up the buildbot stuff?

**Comment by werebus** [comment]
I think we can delete the branch now. If we revert to Travis, it'll create the branch again I'm pretty sure.

For the buildbot stuff, yeah, that can wait a bit. But we would want to clean it up eventually.

**Pull request merged**